### PR TITLE
Added Keyboard shortcuts (clear/set read status)

### LINKED ISSUE #7264: Missing Key Bindings
JabRef 5.2--2020-12-24--6a2a512
Windows 10 10.0 amd64
Java 14.0.2

- [ ] **Mandatory**: I have tested the latest development version from http://builds.jabref.org/master/ and the problem persists

This issue is present on the v5.2 stable.

1. Open Edit > Read Status > ...
2. Open Options > Customize key binding > Edit
3. One cannot edit the key bindings present in the menu

It would be great if all the menu options could be bound to a keystroke combination

image
image

### PR DESCRIPTION
**Pull request by Shadow-Devil**
Added Keyboard shortcuts with defaults for 
 - Clear read status (alt+C)
 - Set read status to read (alt+R)
 - Set read status to skimmed (alt+S)

Fixes #7264

- [x] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

**Review by Siedlerchr — APPROVED** [review]
Thanks for your contribution

**Comment by Shadow-Devil** [comment]
ups, thought i had to close the pull request but i think i have to wait until you merge it.

File: src/main/java/org/jabref/gui/keyboard/KeyBinding.java
Comment (koppor) [inline src/main/java/org/jabref/gui/keyboard/KeyBinding.java:109]: I am not sure whether this is a good default.

Typically, in an application, there are shortcut keys for menu entries. They are typically used by <kbd>Alt<kbd> and then the letter. This is the same pattern than you propose. See also https://www.howtogeek.com/366147/how-to-make-windows-10-underline-and-highlight-menu-shortcut-keys/.

See for instance Excel:

grafik

We are trying to bring them back into JabRef.

Could you please think of other shortcuts? Maybe Ctrl+Alt+X?

**Review by koppor — CHANGES_REQUESTED** [review]

File: src/main/java/org/jabref/gui/keyboard/KeyBinding.java
Comment (Shadow-Devil) [reply src/main/java/org/jabref/gui/keyboard/KeyBinding.java:109]: okay, how about:
- clear: ctrl+alt+x
- read: ctrl+alt+r
- skimmed: ctrl+alt+d   (ctrl+alt+s is "Save all")

Comment (tobiasdiez) [reply src/main/java/org/jabref/gui/keyboard/KeyBinding.java:109]: I think it would also be fine to have no default values.

**Review by tobiasdiez — APPROVED** [review]
LGTM!

**Comment by calixtus** [comment]
If everybody is happy, I'll think I merge. Thanks! 😄

**Pull request merged**
